Sources/SPIManifest/Manifest.swift

```diff
@@ -51,15 +51,21 @@ public struct Manifest: Codable, Equatable {
             public var image: String?
             public var scheme: String?
             public var target: String?
+
+            /// Define a list of targets for which documentation should be generated. The target order determines their display order and the first target is the one displayed if no target is explicitly selected by the user.
             public var documentationTargets: [String]?
 
+            /// Define custom parameters that will be appended to the `package generate-documentation` invocation during the documentation generation process.
+            public var generateDocumentationParameters: [String]?
+
             enum CodingKeys: String, CodingKey {
                 case platform
                 case swiftVersion = "swift_version"
                 case image
                 case scheme
                 case target
                 case documentationTargets = "documentation_targets"
+                case generateDocumentationParameters = "generate_documentation_parameters"
```

Comment:
Or `custom_docgen_parameters`?

Reply:
`documentation_targets` and `custom_documentation_parameters`? The generate is implicit, I think.

Reply:
Yeah, that's better 👍